docs/filesystems/Filesystems.md

```diff
@@ -22,10 +22,10 @@ filesystems {
             url = "https://martha-url-here"
             # The number of times to retry failures connecting or HTTP 429 or HTTP 5XX responses, default 3.
             num-retries = 3
-            # How long to wait between retrying HTTP 429 or HTTP 5XX responses, default 30 seconds.
-            wait-initial = 30 seconds
-            # The maximum amount of time to wait between retrying HTTP 429 or HTTP 5XX responses, default 5 minutes.
-            wait-maximum = 5 minutes
+            # How long to wait between retrying HTTP 429 or HTTP 5XX responses, default 10 seconds.
+            wait-initial = 10 seconds
+            # The maximum amount of time to wait between retrying HTTP 429 or HTTP 5XX responses, default 30 seconds.
+            wait-maximum = 30 seconds
```

Comment:
Again, 30 seconds is short for an overloaded server to crash and start back up. And I'm not convinced a lot of DRS implementations are load balanced / high availablity.